packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -125,6 +121,7 @@ class CupertinoPicker extends StatefulWidget {
     @required this.onSelectedItemChanged,
     @required IndexedWidgetBuilder itemBuilder,
     int childCount,
+    this.magnifier,
```

Comment:
The general pattern is to have a default value here (e.g. `this.magnifier = const CupertinoPickerDefaultMagnifier()`) rather than filling it in in the state. This way, you can actually specify null as an intended value (to not draw anything there if users want).

Reply:
This was my mistake, thank you, and is now resolved.

Reply:
It's not a mistake :) We didn't really document this pattern clearly.

Reply:
But this is a best way.